### PR TITLE
[STORM-1370] - Bug fixes for MultitenantScheduler

### PR DESCRIPTION
**Pull request by jerrypeng**

### CODE REVIEW

**Comment by jerrypeng** [comment]
@d2r can you take a look thanks!

File: storm-core/src/jvm/backtype/storm/scheduler/multitenant/Node.java [hunk 1]
```diff
@@ -334,7 +334,7 @@ public static int countTotalSlotsAlive(Collection<Node> nodes) {
   public static final Comparator<Node> FREE_NODE_COMPARATOR_DEC = new Comparator<Node>() {
     @Override
     public int compare(Node o1, Node o2) {
-      return o2.totalSlotsFree() - o1.totalSlotsFree();
+      return o1.totalSlotsUsed() - o2.totalSlotsUsed();
```
Comment (d2r) [inline storm-core/src/jvm/backtype/storm/scheduler/multitenant/Node.java]: Yes, this was the fix for spreading components on isolated nodes when certain workers are having trouble coming up.

**Comment by jerrypeng** [comment]
unrelated travis failure only for java 8 storm-core build

**Comment by revans2** [comment]
+1 looks good to me.

**Pull request merged**